app/javascript/functionalities.js

```diff
@@ -75,3 +75,31 @@ window.onclick = function(event) {
   hideMenus(event);
   hideNotifications(event);
 };
+
+const showDni = function() {
+  select_location = document.getElementById('user_role')
+  if(select_location.value == 'host') {
+     document.getElementById('hidden_div').style.display = 'block';
+    }
+   else {
+      document.getElementById('hidden_div').style.display = 'none';
+    }
+ };
+
+ setVerified = (user) => {
+    var csrf = document.querySelector('meta[name="csrf-token"]').content;
+    var xhr = new XMLHttpRequest();
+    xhr.open("PUT", "users/"+ user);
+    xhr.setRequestHeader('X-CSRF-Token', csrf);
+    xhr.send();
+    xhr.onload = function() {
+        if (xhr.status != 204) {
```

Comment:
The status is 204 or 200?

Reply:
This might be used, for example, In this case a [PUT] request would be used to save the page, and the 204 No Content response would be sent to indicate that the editor should not be replaced by some other page. I´m updating a value with a checkbox. Don't need to change the view.